spec/models/ordergroup_spec.rb

```diff
@@ -25,6 +25,22 @@
     result = Ordergroup.include_transaction_class_sum.where(id: og).first
     expect(result["sum_of_class_#{ftc1.id}"]).to eq 4
     expect(result["sum_of_class_#{ftc2.id}"]).to eq 440
+    expect(og.account_balance).to eq 444
+
+    ftc1.reload
+    ftc1.update_attributes!(ignore_for_account_balance: true)
+
+    og.reload
+    expect(og.account_balance).to eq 440
+
+    ftt2.reload
+    ftt2.update_attributes!(financial_transaction_class: ftc1)
+
+    og.reload
+    result = Ordergroup.include_transaction_class_sum.where(id: og).first
+    expect(result["sum_of_class_#{ftc1.id}"]).to eq 44
+    expect(result["sum_of_class_#{ftc2.id}"]).to eq 400
+    expect(og.account_balance).to eq 400
```

Comment:
Can we make this separate tests, please? So that we can more easily see which parts break when modifying code.

Reply:
done